### PR TITLE
Jetty 10.0.x fix dependency in jetty-all

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1257,6 +1257,9 @@
     </profile>
     <profile>
       <id>ci</id>
+      <modules>
+        <module>aggregates/jetty-all</module>
+      </modules>
```
Comment (sbordet) [inline pom.xml]: @olamy so this module was only built on CI, not during the release.
Why did you have problems during the release then?

Comment (olamy) [reply pom.xml]: nope the only way. It was only build during release and not on ci :)
so now it will be check on ci

Comment (olamy) [reply pom.xml]: failure validated here :) https://jenkins.webtide.net/job/jetty.project/job/jetty-10.0.x_fix_dependency/2/

Comment (sbordet) [reply pom.xml]: Doh!

**Pull request merged**